README.md

```diff
@@ -1 +1,34 @@
-# cobda-web
\ No newline at end of file
+**Cobda** is a community-like e-commerce website for selling and buying streetwear stuff. Plus, it's accessible anywhere and anytime through the internet.
+
+## Prerequisite
+
+You must download the following to be able to run.
```

Comment:
I don't think this sentence is necessary this the title already has its own meaning.